tekton/tasks/qeb-hwt-task.yaml

```
@@ -0,0 +1,105 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: qeb-hwt
```

```suggestion
  name: qeb-hwt-task
```
The same name which has been referred by taskruns via taskref.